microcontroller/src/rtt.c

```diff
@@ -0,0 +1,18 @@
+#include "rtt.h"
+
+size_t RTT_write(const uint8_t *bp, size_t n){
```

Comment:
It is up to you but it would be good if you insert a space between ') {' (because this is the style I am using)